DER/DerivativesContracts/DerivativesBasics.rdf

```diff
@@ -412,6 +412,20 @@
 		<skos:definition xml:lang="en">specifies the final date on which a delivery notice can be issued</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-bsc;hasPayingParty">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
+		<rdfs:label>has paying party</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-der-drc-bsc;PayingParty"/>
+		<skos:definition>defines the payer party for some aspect of a derivative, such as for a funding leg of a swap</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-drc-bsc;hasReceivingParty">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
+		<rdfs:label>has receiving party</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-der-drc-bsc;ReceivingParty"/>
+		<skos:definition>defines the recipient for some aspect of a derivative, such as for a funding leg of a swap</skos:definition>
```

Comment:
see previous comment. Plus I don't think "receiving" as a term is a good opposite of "paying". Why not payer and payee?
Finally we need to more clearly in the ontology separate paying for the option from paying on exercise. e.g. BUYING (and hence being payer) for a Put option gives you the right to SELL (and be the payee) the underlying Instrument. Are you then the paying or the receiving party?  
Another consideration we haven't discussed: the notion of Assignment - which is what happens when the issuer is required to sell/buy the underlying when the owner of an option exercises it.
Finally, I'm concerned that for exchange traded options, there is no recorded pairing of buyer and seller - it's all accounted for by the market maker.

Reply:
See above - I've replaced this with hasSeller from FND, so this property is no longer needed.  

The points you are raising on the Options ontology, rather than on Derivative Basics, are something we should address in the next round of revisions, though.  We need to include restrictions, and likely new properties, to distinguish the buyer/seller of the underlying asset.  The class for underlying asset is there, but some refactoring to make clear what we are talking about is still needed, as well as dealing with the distinction regarding the market maker.